00_Core/Scripts/Source/MilkMCM.psc

```diff
@@ -3231,7 +3231,7 @@ endState
 state Debug_MM_MilkMax_Basevalue_Slider
 	event OnSliderOpenST()
 		SetSliderDialogStartValue(MME_Storage.getMilkMaxBasevalue(MaidlistA[MaidIndex]))
-		SetSliderDialogDefaultValue(0)
+		SetSliderDialogDefaultValue(2)
 		SetSliderDialogRange(0, 20)
```

Comment:
This is one of 3 parameters the user can specify to calculate a customized maximum value. The formula is `(MilkBasevalue + Level*MilkScalefactor)*BreastCount'`. The safeguard in updateMilkMaximum() guarantees that whatever the user selects for MilkBasevalue, MilkScalefactor and BreastCount the resulting minimum value for MilkMax will never be less then 1.

Possible scenarios:
- MilkBasevalue = 2, MilkScalefactor = 1, BreastCount = 2   => this is the default 
- MilkBasevalue = 20, MilkScalefactor = 0, BreastCount = 2   => MilkMax is fixed to 20
- MilkBasevalue = 20, MilkScalefactor = 0, BreastCount = 4   => MilkMax is fixed to 40 (e.g. for a character model with 2 breast rows?)
- MilkBasevalue = 20, MilkScalefactor = -1, BreastCount = 2 => MilkMax decreases from 20 to 1 with each level up (maybe to model exhaustion due to continued milking?)

For the last scenario it would even be reasonable to allow MilkMax = 0 and deny any further milking (character has 'run dry'?). But since this breaks the current code it's better to limit to a minimum of 1 instead.